src/layers/models/layersManager.ts

```diff
@@ -150,14 +151,10 @@ export class LayersManager {
 
     const newCache: IMapProxyCache = {
```

Comment:
refactor addLayer and Update layer to use one function for creating newCache and newLayer -> check if other code can be refactored

Reply:
fixed in 0b9e76c